### ISSUE TITLE
Showing Hiding object documentation missing

### ISSUE THREAD

**Issue by Scibearspace** [senshu/Sozi#354]
Hi,

This seems like a nice program. However sometimes the documentation seems to leave a new user in the dark.

Example, whilst trying to work out how to show / hide an object I get the error message.

This page is part of the documentation for Sozi 13. As we are currently in the process of releasing Sozi 15, the content of this page is considered obsolete and will be updated soon.

It would be usefull to have a link to the old docs at the very least ~ someone may then update them for you.
But not having the details makes it hard for a new user to decide if the function is even available, which is a very common function in presentation software.

Link to the specific page
http://sozi.baierouge.fr/pages/tutorial-showing-hiding.html

**Comment by senshu** [comment]
> sometimes the documentation seems to leave a new user in the dark

Yes. I hope I can find time to complete the docs.

> It would be usefull to have a link to the old docs at the very least

What do you mean? As the message explains, that page is part of the "old docs". However, it is useless if you use Sozi 15 or 16.

**Comment by Scibearspace** [comment]
sorry, i've been away, and then very busy with my thesis !

If you give access to the old docs, it may be possible that a user may be able to figure out what is needed to be done ? or has it really changed that much ?

Also it would give the user the option of using the old version.

The main issue for me is that I can't even find the old docs, as I would try to use the old method to see if it still worked, I may even write a small html for the doc page once I've better understood how to perform my desired function.

**Comment by senshu** [comment]
> If you give access to the old docs, it may be possible that a user may be able to figure out what is needed to be done ? or has it really changed that much ?

Actually, showing and hiding objects has never been officially supported in Sozi 13. I planned to implement it, so I added a page to the web site about this feature but it has always been empty.

For Sozi 13, I have proposed [a workaround that several users found useful](https://github.com/senshu/Sozi/issues/57).

In Sozi 15, you are supposed to use layers to achieve a similar effect.
See the files `using-layers.*` in the [samples folder of the source repository](https://github.com/senshu/Sozi/tree/master/samples).

> Also it would give the user the option of using the old version.

You still have the option, but that version is no longer supported. It means that I am not willing to fix bugs, implement new features, write docs, or answer questions about it any more.
Remember that I am the only developer of Sozi and I do that in my spare time.
